scrapi/harvesters/datacite.py

```diff
@@ -0,0 +1,38 @@
+'''
+Harvester for the DataCite MDS for the SHARE project
+
+Example API call: http://oai.datacite.org/oai?verb=ListRecords&metadataPrefix=oai_dc
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+from scrapi.base.helpers import updated_schema, oai_extract_dois
+
+
+class DataciteHarvester(OAIHarvester):
+    short_name = 'datacite'
+    long_name = 'DataCite MDS'
+    url = 'http://oai.datacite.org/oai'
+
+    base_url = 'http://oai.datacite.org/oai'
+    property_list = ['date', 'identifier', 'setSpec', 'description']
+    timezone_granularity = True
+
+    @property
+    def schema(self):
+        return updated_schema(self._schema, {
+            "description": ("//dc:description/node()", get_second_description),
+            "uris": {
+                "canonicalUri": ('//dc:identifier/node()', oai_extract_dois),
+                "objectUris": ('//dc:identifier/node()', oai_extract_dois)
+            }
+        })
+
+
+def get_second_description(descriptions):
```

Comment:
Each entry pretty reliably has 2 description fields, the first of which is a category of some kind, and the second is usually an actual description. You can [see a sample query](http://oai.datacite.org/oai?verb=ListRecords&metadataPrefix=oai_dc) or an example here:

![screen shot 2015-06-30 at 3 34 43 pm](https://cloud.githubusercontent.com/assets/801594/8440049/d4258bfa-1f3d-11e5-8b6b-251fa029f5e6.png)